lib/nxt_state_machine/graph.rb

```diff
@@ -0,0 +1,75 @@
+module NxtStateMachine
+  class Graph
+    def initialize(state_machine, **options)
+      @state_machine = state_machine
+      @options = default_options.merge(**options)
+    end
+
+    def draw
+      require 'ruby-graphviz'
```

Comment:
The `require` should not be within the method, should it?

Reply:
It is. I only wanted to require in case someone actually tries to draw the graph and not have this as a dependency if you do not use it.